src/sbt-test/sbt-typesafe-conductr/sbt-typesafe-conductr-tester/version.properties

```diff
@@ -1 +0,0 @@
-../../../../version.properties
```

Comment:
I really don't get why we went to this property based approach @ktoso - it is more complex than what we have in other plugins such as sbt-bundle.